### PR TITLE
Parity list added

### LINKED ISSUE #83: List Parities
## Prerequisite

- [x] I am using the latest version of the code, and checked the documentation related to my needs.

* **I'm submitting a ... (remove the irrelevant one)**
  - [x] feature request

* **Creating functionality for API Requirement 1.2.3.**

* **Deadline for resolution: 3 May**

### PR DESCRIPTION
**Pull request by buseg**
# Pull Request Template

## Description

Created API end to return parities included in the database. 

Fixes #83

## Type of change

Please delete options that are not relevant.

- [x] New feature (new functionality)

## How Has This Been Tested?

Please describe the tests that you ran. 
If you haven't committed related tests, please share the related issue requesting tests for this functionality.

- [ ] Test 1
- [ ] Test 2

Test Issue # (issue)

## Checklist:

- [x] My code follows pep8
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, or created a related issue.
- [ ] I have committed tests that prove my fix is effective or that my feature works. Or, I have opened an issue.
- [x] I have added 2 reviewers to get the approval of.

### CODE REVIEW

File: practice-app/api/views.py [hunk 1]
```diff
@@ -14,3 +16,23 @@ def post(self, request):
         return Response({
             'message': f'Hello, {name}.'
         })
+
+class ParityListView(APIView):
+    def get(self, request):
+        parities = Parity.objects.all()
+        parity_names = []
+
```
Comment (fatih-iver) [inline practice-app/api/views.py]: You may use a set instead of a list in the first place.

File: practice-app/api/views.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
+from api.models import Parity
+import json
```
Comment (mertyg) [inline practice-app/api/views.py]: where do you use the json package? this line seems redundant to me

File: practice-app/api/urls.py [hunk 3]
```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path('helloworld/', views.HelloWorldView.as_view(), name='helloworld'),
-]
\ No newline at end of file
+    path('paritylist/', views.ParityListView.as_view(), name='paritylist'),
```
Comment (mertyg) [inline practice-app/api/urls.py]: as the url path, we planned to stick with this structure:
'/parity/' for listing the parities,
'/parity/latest/' for the latest data, '/parity/history' etc. . Hence, you might want to change /paritylist/ for the sake of consistency

**Comment by buseg** [comment]
Can you review the latest version @mertyg @fatih-iver I think it can be merged now?

**Pull request merged**